oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -106,6 +106,22 @@ public static GoogleCredentials fromStream(InputStream credentialsStream, HttpTr
   }
 
   /**
+   * Default constructor.
+   **/
+  public GoogleCredentials() {
+    this(null);
+  }
```

Comment:
The prevailing convention from the Java V1 libraries was to have them after factories.